package.json

```diff
@@ -127,5 +127,6 @@
     "ts-jest": "27.1.4",
     "ts-node": "9.1.1",
     "typescript": "5.0.2"
-  }
+  },
+  "packageManager": "yarn@1.22.19+sha1.4ba7fc5c6e704fce2066ecbfb0b0d8976fe62447"
```

Comment:
any reason of adding it?

Reply:
I did yarn and it got added on it's own. Lemme remove it

Reply:
trying removing and then do yarn

Reply:
yes it came up again but I removed it before pushing. Might be some configuration for me. will check it out